core/runtime/src/main/java/io/quarkiverse/flow/recorders/WorkflowDefinitionRecorder.java

```diff
@@ -34,4 +38,16 @@ public Supplier<WorkflowDefinition> workflowDefinitionSupplier(String flowDescri
         };
     }
 
+    public Supplier<WorkflowDefinition> workflowDefinitionFromFileSupplier(String location) {
```

Comment:
I thing this String parameter can be a patch parameter, you are later doing Paths.get(location)